samples/ChangeFeedProcessor/DocumentDB.ChangeFeedProcessor/DocumentDB.ChangeFeedProcessor.csproj

```diff
@@ -38,13 +38,11 @@
     <StartupObject />
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Azure.Documents.Client, Version=1.13.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\packages\Microsoft.Azure.DocumentDB.1.13.1\lib\net45\Microsoft.Azure.Documents.Client.dll</HintPath>
-      <Private>True</Private>
+    <Reference Include="Microsoft.Azure.Documents.Client, Version=1.17.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
Just to be consistent, shouldn't this Reference also remove the Version, Culture, PublicKeyToken, processorArchitecture removed similar to how you did in ChangeFeedMigrationSample.csproj?

I'd stick to whatever VS does for adding the reference to the assembly when you import the nuget package.